.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,100 @@
+name: Report an issue
+description: Create a bug report to fix an issue
+title: 'Issue: '
+labels: ['bug']
+body:
+  - type: markdown
+    attributes:
+      value: >
+        :warning: **Please do not report security vulnerabilities here**
+
+        Be careful with sensitive information and security vulnerabilities. In order to report bugs that could contain
+        sensitive information, use [Launchpad](https://bugs.launchpad.net/ubuntu/+source/ubuntu-insights/+filebug) instead.
+        On Ubuntu machines, you can use `ubuntu-bug ubuntu-insights` to collect relevant information.
+
+
+        Thanks for taking the time to report an issue and help improve Ubuntu Insights! Please fill out the form below as
+        best as you can so that we can help you.
+
+        Your additional work here is greatly appreciated and will help us respond as quickly as possible. For general
+        support or usage questions, use [Ubuntu Discourse](https://discourse.ubuntu.com/c/desktop/8).
```

Comment:
Links to Desktop's discourse, is this the intention?

Reply:
These were copied from a template from the Desktop team. I'm unsure as to where else to link it to, but given that WSL and Multipass are both under Desktop and there are Desktop components, it is probably fine.